msgid ""
msgstr ""
"Project-Id-Version: openplotter\n"
"POT-Creation-Date: 2019-11-23 16:44+0100\n"
"PO-Revision-Date: 2019-12-16 13:01+0100\n"
"Last-Translator: \n"
"Language-Team: Spanish\n"
"Language: es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 1.8.7.1\n"
"X-Poedit-Basepath: ../../..\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Crowdin-Project: openplotter\n"
"X-Crowdin-Language: es-ES\n"
"X-Crowdin-File: openplotter-network.po\n"
"X-Poedit-SearchPath-0: .\n"

#: networkPostInstall.py:30
msgid "Editing config files..."
msgstr "Editando archivos de configuración..."

#: networkPostInstall.py:49
msgid "DONE"
msgstr "HECHO"

#: networkPostInstall.py:51 networkPreUninstall.py:49
msgid "FAILED: "
msgstr "ERROR: "

#: networkPreUninstall.py:29
msgid "Restoring network Debian defaults..."
msgstr "Restaurando los valores predeterminados de red del sistema..."

#: networkPreUninstall.py:40
msgid "Removing openplotter-network service..."
msgstr "Eliminando servicio de Openplotter-red..."

#: networkPreUninstall.py:48
msgid "DONE. PLEASE REBOOT TO RESTORE NETWORK SETTINGS"
msgstr "Hecho. POR FAVOR REINICIE PARA RESTAURAR AJUSTES DE RED"

#: openplotterNetwork.py:36
msgid "OpenPlotter Network"
msgstr "Red OpenPlotter"

#: openplotterNetwork.py:46
msgid "Help"
msgstr "Ayuda"

#: openplotterNetwork.py:49
msgid "Settings"
msgstr "Ajustes"

#: openplotterNetwork.py:52
msgid "Addresses and Ports"
msgstr "Direcciones y Puertos"

#: openplotterNetwork.py:54
msgid "Check Network"
msgstr "Comprobar red"

#: openplotterNetwork.py:57
msgid "Install Wifi Drivers"
msgstr "Instalar contr. Wifi"

#: openplotterNetwork.py:64
msgid "Access Point"
msgstr "Punto de acceso]"

#: openplotterNetwork.py:132
msgid "Network Mode"
msgstr "Modo de red"

#: openplotterNetwork.py:137
msgid "AP"
msgstr "PA"

#: openplotterNetwork.py:147
msgid "5 GHz"
msgstr "5GHz"

#: openplotterNetwork.py:148
msgid "Add ethernet port to the AP"
msgstr "Añadir puerto ethernet al AP"

#: openplotterNetwork.py:170
msgid "Access Point Settings"
msgstr "Ajustes de punto de acceso"

#: openplotterNetwork.py:174
msgid "Sharing Internet device"
msgstr "Compartiendo internet desde"

#: openplotterNetwork.py:182
msgid ""
"SSID \n"
"maximum 32 characters"
msgstr ""
"SSID \n"
"máximo 32 caracteres"

#: openplotterNetwork.py:190
msgid ""
"Password \n"
"minimum 8 characters required"
msgstr ""
"Contraseña\n"
"mínimo 8 caracteres y robusta"

#: openplotterNetwork.py:200
msgid "Channel"
msgstr "Canal"

#: openplotterNetwork.py:315 openplotterNetwork.py:336
#: openplotterNetwork.py:342 openplotterNetwork.py:399
#: openplotterNetwork.py:438 openplotterNetwork.py:459
#: openplotterNetwork.py:608
msgid "none"
msgstr "nada"

#: openplotterNetwork.py:406
msgid "Wait please... "
msgstr "Un momento, por favor... "

#: openplotterNetwork.py:505
msgid "Do you want to change the internet connection directly?"
msgstr "¿Quieres cambiar la conexión a Internet inmediatamente?"

#: openplotterNetwork.py:506 openplotterNetwork.py:584
msgid "Question"
msgstr "Pregunta"

#: openplotterNetwork.py:533
msgid "Your SSID must have a maximum of 32 characters and your password a minimum of 8."
msgstr "SSID de como máximo 32 caracteres y contraseña un mínimo de 8 y robusta."

#: openplotterNetwork.py:573 openplotterNetwork.py:580
msgid "Validated, please reboot"
msgstr "Aplicado, por favor reinicie"

#: openplotterNetwork.py:583
msgid ""
"Changes will be applied after next reboot.\n"
"\n"
"If something goes wrong and you are on a headless system,\n"
"you may not be able to reconnect again.\n"
"\n"
"Are you sure?"
msgstr ""
"Los cambios seran aplicados tras reiniciar.\n"
"\n"
"Si algo falla y esta usando un sistema sin pantalla (escritorio remoto),\n"
"tal vez no pueda volver a conectar.\n"
"\n"
"¿Está seguro?"

#: openplotterNetwork.py:607 openplotterNetwork.py:608
msgid "wifi access point: "
msgstr "punto de acceso wifi: "

#: openplotterNetwork.py:625
msgid "wifi client: "
msgstr "cliente wifi: "

#: openplotterNetwork.py:638
msgid "running"
msgstr "trabajando"

#: openplotterNetwork.py:639
msgid "stopped"
msgstr "parado"

#: openplotterNetwork.py:653
msgid "IP address\n"
msgstr "Dirección IP\n"

#: openplotterNetwork.py:666
msgid "Installing Wifi modules... "
msgstr "Instalando módulos Wifi... "

#: openplotterNetwork.py:668
msgid "Done."
msgstr "Hecho."

#: openplotterNetwork.py:706
msgid "There are conflicts between server connections"
msgstr "Hay conflictos entre las siguientes conexiones del servidor"

#: openplotterNetwork.py:707
msgid "No conflicts between servers connections"
msgstr "No hay conflictos entre las conexiones de servidores"

#: ports.py:28
msgid "VNC Remote Desktop"
msgstr "Escritorio remoto VNC"

#: startup.py:46
msgid "Checking WIFI Access Point password..."
msgstr "Revisando contraseña del punto de acceso wi-fi..."

#: startup.py:68
msgid ""
"Security warning: You are using the default WIFI Access Point password.\n"
"Please change password in OpenPlotter Network."
msgstr "Advertencia de seguridad. Esta usando la contraseña basica del punto de aceso wi-fi. Por favor, cambiela por una mas robusta en OpenPlotter > red."

#: startup.py:69
msgid "changed"
msgstr "cambiado"
